packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -234,6 +238,12 @@ const PayableDetailsFormBase = forwardRef<
 
     const className = 'Monite-PayableDetailsForm';
 
+    const ocrFields = useOcrFields();
+
+    useEffect(() => {
+      trigger();
```

Comment:
I wouldn't want to pull validation for all fields on a form as soon as we go to it without additional conditions.

Reply:
My intention here is to trigger this action so that it highlights any fields that are missing for the user. This was specifically requested by @arthemium  to ensure proper highlighting when fields are incomplete

Reply:
Discussed directly with @radist2s , we want to trigger only OCR fields not all fields